source/dpaa_shutdown.rst

```diff
@@ -41,7 +41,7 @@ Within a major frame (32.2 seconds), one should see:
 * DPA-A POWER should go to zero
 * 1DP28AVO (DPA-A +28V Input Voltage) is expected to have a small uptick, ~0.5 V, consistent with
   the load suddenly dropping to zero
-* The software and hardware bilevels will likely not have normal values if BEP side A is active.
+* The software and hardware bilevels will likely not have normal values if BEP side A is inactive.
```

Comment:
The original text is correct. The idea here is that BEP side A is the active one, but we have shut off the DPA side A power supply, so the bilevels coming from the BEP cannot be trusted. 

The possibility being referred to here is if DPA side A shut off but BEP B was active instead.

Reply:
Ah, sorry about that.  I had misunderstood and see.

Reply:
@jazan12 Actually, can you change this back to "active"?